qml/plugins/RestartBluetooth.qml

```diff
@@ -0,0 +1,17 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Utilities 1.0
+
+ActionItem {
+    //% "Bluetooth"
+    title: qsTrId("settings_bluetooth-bluetooth")
+    //% "Restart"
+    actionName: qsTrId("sailfish-tools-bt-restart")
```

Comment:
```suggestion
    actionName: qsTrId("sailfish-tools-bt-bluetooth_restart)
```
Maybe something along these lines?

Reply:
This is a weird one.

The qml started out as a copy from RestartNetwork.qml, which has a key `qsTrId("sailfish-tools-bt-restart")` (note the `bt`), see commit 50267f3b.

No objections to change it, but this does not come from me ;)

Reply:
`-bt-` stands for button. Reusing existing translation is not a bad idea at all.

Reply:
Thanks for clarifying. So this one stays, reusing the translation from Network.